librz/bin/format/elf/elf.h

```diff
@@ -186,7 +177,7 @@ struct Elf_(rz_bin_elf_obj_t) {
 	bool relocs_patched;
 	ut64 reloc_targets_map_base;
 
-	RzList /*<RzBinElfNoteSegment>*/ *note_segments;
+	RzVector *notes; // RzVector<RzVector<RzBinElfNote>>
```

Comment:
Please add a documentation comment here clarifying that the elements of the outer `RzVector` each describe an individual segment.

Reply:
I add a comment